src/features/basic/rename-tab.ts

```diff
@@ -0,0 +1,14 @@
+function updateTabTitle() {
+  const screenName = document
+    .querySelector(`.${C.ScreenControls.currentScreenName}`)
+    ?.textContent?.trim();
```

Comment:
You can get the current screen name using this:
```suggestion
import { screensStore } from '@src/infrastructure/prun-api/data/screens';

  const screenName = screensStore.current.value?.name;
```